color-picker/src/commonMain/kotlin/com/godaddy/android/colorpicker/harmony/HarmonyColorPicker.kt

```diff
@@ -33,12 +32,41 @@ import kotlin.math.atan2
 import kotlin.math.hypot
 import kotlin.math.min
 
+@Deprecated(
+    message = "Use version with programmatic updates",
+    replaceWith = ReplaceWith(
+        """HarmonyColorPicker(
+            modifier = modifier,
+            harmonyMode = harmonyMode,
+            value = HsvColor.from(color),
+            onValueChanged = onColorChanged
+        )"""
+    )
+)
 @Composable
 fun HarmonyColorPicker(
     modifier: Modifier = Modifier,
     harmonyMode: ColorHarmonyMode,
     color: Color = Color.Red,
     onColorChanged: (HsvColor) -> Unit
+) {
+    HarmonyColorPicker(
+        modifier = modifier,
+        harmonyMode = harmonyMode,
+        value = HsvColor.from(color),
+        onValueChanged = onColorChanged
+    )
+}
+
+/**
+ * Color Picker that programmatically updates.
```

Comment:
[nit] this comment does not add too much value from a user perspective, the opposite of the comments in ClassicColorPicker.

I suggest removing it(as it was previously) or updating it like the ClassicColorPicker